CMSgov/mint-app
# [NOREF] Remove architect role

# EASI-000

## Changes and Description

- Reverts changes from https://github.com/CMSgov/mint-app/pull/401
  - Re-assigns anyone with the `ARCHITECT` role to `IT_LEAD`
  - Removes `ARCHITECT` as a possible role type

## How to test this change

- Locally, delete the `V68` migration
- Start the app with `scripts/dev up`
- Seed the DB with `scripts/dev db:seed`
- Modify one of the collaborator's role to be `ARCHITECT`
- Recreate the `v68` migration
- Run migrations with `scripts/dev db:migrate`
- Ensure the collaborator was re-assigned the role.

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
